packages/babel-plugin-transform-class-properties/src/index.js

```diff
@@ -36,7 +36,7 @@ export default function({ types: t }) {
       VALUE: value ? value : t.identifier("undefined"),
```

Comment:
we have a method for it `path.scope.buildUndefinedNode(` - btw this should be a linting rule in babel?